ethology/io/compress.py

```diff
@@ -0,0 +1,73 @@
+import os
+import shutil
+
+import ffmpeg
+
+
+class FFmpegNotFoundError(Exception):
+    """Custom exception for missing ffmpeg installation."""
+
+    pass
+
+
+def check_ffmpeg_installed():
+    """Check if ffmpeg is installed by looking for the executable."""
+    if shutil.which("ffmpeg") is None:
+        raise FFmpegNotFoundError(
+            "FFmpeg is not installed or not found in system PATH."
+        )
+    else:
+        return True
```

Comment:
We could do away with this bit, as the user would get a "module not found error" otherwise, which may be clear enough.